contracts/retirement/KlimaCarbonRetirements.sol

```diff
@@ -0,0 +1,199 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.10;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+    @title Klima Retirement Storage
+    @notice This is used to store any offset retirements made through Klima retirement helper contracts.
+ */
+contract KlimaCarbonRetirements is Ownable {
+    struct Retirement {
+        uint256 totalRetirements;
+        uint256 totalCarbonRetired;
+        uint256 totalClaimed;
+        mapping(uint256 => address) retiredPool;
```

Comment:
It would be nice to have natspec in these mappings, explaining the key is `totalRetirements`. It might also have been better to collapse all the different mappings into a single `uint256 => Data` mapping where `Data` is a struct that holds `retiredPool`, `retiredAmount`, `retirementBeneficiary`, and `retirementMessage`.